examples/pod_watcher.rs

```diff
@@ -4,14 +4,16 @@ use k8s_openapi::api::core::v1::Pod;
 use kube::{
     api::{Api, ListParams, ResourceExt},
     runtime::{utils::try_flatten_applied, watcher},
-    Client,
+    Client, Config,
 };
 
 #[tokio::main]
 async fn main() -> Result<()> {
     std::env::set_var("RUST_LOG", "info,kube=debug");
     env_logger::init();
-    let client = Client::try_default().await?;
+    let config = Config::infer().await?;
+    tracing::info!("config is: {:?}", config);
+    let client = Client::try_from(config)?;
```

Comment:
@clux Should I drop this change?

Reply:
ah, whoops, yes please. that was a debug for me while developing.